Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -494,6 +494,7 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
         }
 
         return fileReferences.map {
+            // Should we add platform filters here?
```

Comment:
But which filter do we apply in that case? If we have a graph:

```
A (can contain bundle) -- [filter1] --> B (static library) --[filter2]--> C (bundle)
```

It's not obvious from the Tuist standpoint what the user would expect there. Do we apply the filter 2 when copying the bundle into A? Do we apply filter 1? Do we find the common elements across all the filters and use those?

Reply:
We could just leave this to a later phase when exposing explicit filters to users via the API.